jenkins/ci.opensuse.org/openstack-cleanvm.yaml

```diff
@@ -47,7 +47,23 @@
     builders:
       - update-automation
       - shell: |
-          #!/bin/bash
+          export automationrepo=~/github.com/SUSE-Cloud/automation
+          export jtsync=${automationrepo}/scripts/jtsync/jtsync.rb
```

Comment:
We need a workaround for the cleanvm job (due to old host of cleanvm VM).
Please add:
```
export jtsync_fix="ssh root@tu-sle12"
```